slam-libraries/viam-cartographer/src/slam_service/slam_service.h

```diff
@@ -269,6 +274,10 @@ class SLAMServiceImpl final : public SLAMService::Service {
     size_t current_file_offline = 0;
     std::string current_file_online;
 
+    // Enable Localization mode to utilize a cached version of the pointcloud
+    // map, to skip unecessary encoding steps
+    std::atomic<bool> localization_map_ready{false};
+
```

Comment:
[minor]

```suggestion
```

Reply:
was already making changes so I added this to my commit